### ISSUE TITLE
Load external savefiles into mGBA 1-4 save slots

### ISSUE THREAD

**Issue by Berbe** [mgba-emu/mgba#3262]
Version: `0.10.3`
This is related to #1072.

Here is the steps I followed:
1. `File` > `Load ROM…` <- Loaded game ROM
2. `File` > `Save games` > `Use player 1 save game`
3. `File` > `Save games` > `Load alternate save game…` <- Loaded external `.sav` file
4. Continued the game and used the in-game save feature
5. Restarted mGBA: player 1 save game still selected
No save game loaded

I also tried:
* another slot
* the `Load temporary save game…` option

to no avail

What am I doing wrong?

**Comment by endrift** [comment]
Load alternate save game does not overwrite the local .sav file; it writes back to the external one.

**Comment by Berbe** [comment]
What is the difference between the alternate and the temporary option then?
Is there a way to load an external save to one of the mGBA profiles?

**Comment by endrift** [comment]
With temporary save it's not written back at all. To load it into an mGBA profile you just overwrite the relevant file (.sav, sa2, .sa3, sa4 in the save directory, by default the same directory as the ROM)

**Comment by Berbe** [comment]
Thanks for those explanations!
Do you think they could land in some documentation and/or contextual help?

Another idea would be to unify some menu entries so it makes clear profiles are inactive when you load a temporary or an alternate save game.
Some feedback (window title? status bar?) telling which profile and or external save and whether/where save files are being overwritten on save could also help.

TL; DR: Some UX hints would be welcome, so #1072 or this issue would not happen anymore.

What do you think about it?
I am no developer, but if I can be of any help, I would gladly jump on board!
